### ISSUE TITLE
train a support vector machine model

### ISSUE THREAD

**Issue by jacob-umich** [jacob-umich/stats-dash#12]
we need to try 3 types of models for our machine learning model. svm works well for small data. Using the training data from get_data, use the first 3 columns to predict the last column. Info on how to train a svm model is here

https://scikit-learn.org/stable/modules/svm.html#regression

You need to create a function that takes in the data and model args, then returns the trained model

```
def train_svm(training, kernel, etc):
# train the model
return model
```

**Comment by tsivitse** [comment]
I added the svm to the train_model.py. It's giving what i believe are % accuracy numbers for the 4th column so I think it's working

**Comment by jacob-umich** [comment]
@tsivitse the svm page recommends scaling the data for this model. Since all of our data are percentages, i am going divide all the data by 100

**Comment by jacob-umich** [comment]
just kidding. Apparently the models don't work like that. When all the values were around 1, every model made the same prediction
